test/error-placement.js

```diff
@@ -162,10 +162,10 @@ test( "errorcontainer, show/hide only on submit", function() {
 	ok( labelContainer.is( ":visible" ), "must be visible" );
 	ok( container.is( ":visible" ), "must be visible" );
 
-	$( "#firstname" ).val( "hix" ).keyup();
```

Comment:
I agree, but you have to know that I changed the attachment of the event in `init()` to use `input` event instead of `keyup` if the former is supported, and to use the latter otherwise. Should I change the logic ?